Update README.md

alanyee opened the pull request:
Update readability in example, replaced HTTP link to HTTPS links, and added a link

alanyee committed:
Update README.md

Update readability in example, replaced HTTP link to HTTPS links, and added a link

tensorflow-jenkins commented:
Can one of the admins verify this patch?

wolffg commented on README.md:
```diff
@@ -55,16 +55,17 @@ $ python
 * [TensorFlow MOOC on Udacity](https://www.udacity.com/course/deep-learning--ud730)
+* [TensorFlow course at Stanford](https://web.stanford.edu/class/cs20si)
 
 The TensorFlow community has created amazing things with TensorFlow, please see the [resources section of tensorflow.org](https://www.tensorflow.org/about/#community) for an incomplete list.
```
While you're at it, https://www.tensorflow.org/community/ is at better link than the about link here.

alanyee replied:
I have done so, and in doing so, I realized that an above link had the same link as the link you suggested. So I have made a few more edits since the above said link seems outdated.

wolffg reviewed (approved):
Approved, but maybe change one more link.

alanyee committed:
Update README.md

-Changed link as suggested
-Added links and replaced some more
-Few other minor edits

caisq commented on README.md:
```diff
@@ -25,12 +25,12 @@ variety of other domains, as well.
-tracking requests and bugs, but please see
-[Community](https://www.tensorflow.org/community/) for general questions
-and discussion.**
+tracking requests and bugs. So please see the 
```
Superfluous "the"

caisq commented on README.md:
```diff
@@ -25,12 +25,12 @@ variety of other domains, as well.
-[Community](https://www.tensorflow.org/community/) for general questions
-and discussion.**
+tracking requests and bugs. So please see the 
+[TensorFlow discuss](https://groups.google.com/a/tensorflow.org/forum/#!forum/discuss) for general questions
```
nit: capitalize the first letter of "discuss"

caisq commented on README.md:
```diff
@@ -55,16 +55,17 @@ $ python
+* [TensorFlow course at Stanford](https://web.stanford.edu/class/cs20si)
 
-The TensorFlow community has created amazing things with TensorFlow, please see the [resources section of tensorflow.org](https://www.tensorflow.org/about/#community) for an incomplete list.
+Learn more about the TensorFlow community at the [community page of tensorflow.org](https://www.tensorflow.org/about/community) for a few ways on how to participate.
```
a few ways to participate

alanyee committed:
Update README.md

Applying requested changes

alanyee committed:
Merge branch 'master' into patch-1

alanyee commented:
@wolffg Is there anything else I should change, or can this be merged now?

wolffg committed:
Update README.md

wolffg commented:
I added a PR to change the last link to https://www.tensorflow.org/community/welcome, as the other link 404s.

alanyee committed:
Update README.md

Update link

alanyee committed:
Update README.md

Remove extra line of white space

jhseu commented:
Jenkins, test this please
